### PR TITLE
Basic layout for frontpage content

### PR DESCRIPTION
**Pull request by visitharri**

### CODE REVIEW

**Comment by visitharri** [comment]
Still working with the unit test, so don't merge yet

**Comment by visitharri** [comment]
I think the unit test is failing because of fetching problems of mock-backend data during the test. Or is it because in some situations, when backend does not work, data might be undefined. Should there be some check-ups in the code that the data is not undefined. What do you think?

File: frontend/src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -1,12 +1,152 @@
...
     return (
-        <div>
-            <h1>{"Here is the page's content"}</h1>
-            <GeocacheList/>
-        </div>
+        <>
+            <div className="screen">
+                <div className="main">
+                    <Container>
+                        <Row><div className="mx-auto" style={{ width: "auto" }}>
```
Comment (kemm0) [inline frontend/src/pages/HomePage/HomePage.tsx:27]: This is not a proper way to render a collection of items. What if the amount of geocaches we want to display changes? It shouldn't be hard-coded like this. Instead, a [mapping function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) should be used.

    return (
        <div>
            <h3>Uusimmat geokätköt</h3>
            <Container>
                {
                    geocaches.map((cache) => {
                        return (
                            <Row key={cache.id}>
                                <Card>
                                    <div>{cache.name}</div>
                                    <div>{cache.placedDate}</div>
                                </Card>
                            </Row>
                        )
                    })
                }
            </Container>
        </div >
    );

Comment (kemm0) [inline frontend/src/pages/HomePage/HomePage.tsx:27]: Also, the styles should be in the scss file, not in this file

File: frontend/src/pages/MapViewPage/MapViewPage.scss [hunk 2]
```diff
@@ -1,6 +1 @@
 @import url("../../styles/common.scss");
-@import "leaflet/dist/leaflet.css";
```
Comment (kemm0) [inline frontend/src/pages/MapViewPage/MapViewPage.scss:2]: Why is this removed?

**Review by kemm0 — CHANGES_REQUESTED** [review]
I left some comments where changes should be made. Also you should pull the changes from main to your own branch, to check for any merge conflicts. It seems your branch is 6 commits behind main. The issue that you were having with the backend has been solved on main branch, by mocking the backend response in GeoCacheList.test.tsx.

File: frontend/src/pages/MapViewPage/MapViewPage.scss (hunk 2 above)
Comment (visitharri) [reply frontend/src/pages/MapViewPage/MapViewPage.scss:2]: This is double code, which is already in map component's scss file. When I refactored map into components, we talked about it.

File: frontend/src/pages/HomePage/HomePage.tsx (hunk 1 above)
Comment (visitharri) [reply frontend/src/pages/HomePage/HomePage.tsx:27]: Okay, this happens, when someone is copy pasting things from the web.

Comment (visitharri) [reply frontend/src/pages/HomePage/HomePage.tsx:27]: Yes, that looks better. I'll try that

File: frontend/src/pages/MapViewPage/MapViewPage.scss (hunk 2 above)
Comment (kemm0) [reply frontend/src/pages/MapViewPage/MapViewPage.scss:2]: Ok, I forgot about that :D

File: frontend/src/pages/HomePage/HomePage.tsx (hunk 1 above)
Comment (Mel-dev-hub) [reply frontend/src/pages/HomePage/HomePage.tsx:27]: Isn't this the same logic of GeocacheList component? Both Home page and GeocacheList are doing the same thing but with a different look. So we could either get rid of GeocacheList or update it and reuse it.

**Review by kemm0 — APPROVED** [review]
